modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -33,7 +39,7 @@ components:
                     type: string
                 id:
                     description:
-                        The asset library's ID.
+                        The asset library's group ID.
```

Comment:
Shouldn't this be named `groupId` now for consistency with AssetLibraryBrief?

Reply:
answered on the next comment

Reply:
ah!, and NO, on headless is `siteId` no `groupId`!